src/styles/core/_mixins.scss

```diff
@@ -31,3 +31,77 @@ aka functions to output css
   -o-transition: $args;
   transition: $args;
 }
+
+@mixin button-variant($color, $background, $border) {
+  color: $color;
+  background-color: $background;
+  border-color: $border;
+
+  &:focus,
+  &.focus {
```

Comment:
These should be .is-focus, .is-active, etc. for the class versions to match SUIT naming.